src/app/pages/HomePage/Graph/ParaTimeSelector/index.tsx

```diff
@@ -160,6 +160,12 @@ interface ParaTimeSelectorProps extends ParaTimeSelectorBaseProps {
 
 const localStore = storage()
 
+// This is a special layer used to indicate that we should zoom out,
+// and see the whole universe.
+export const UniverseLayer = 'Universe'
```

Comment:
Don't name it "layer"

Reply:
It impacts downstream things like `setSelectedLayer` too

Reply:
OK, what naming would y'all suggest, instead of layer? `section` ? `region` ? `part` ? `segment` ? Something else entirely?

Reply:
I would name it area :shrug:

A similar concept in SVG: https://developer.mozilla.org/en-US/docs/Web/SVG/Element/view
and HTML https://developer.mozilla.org/en-US/docs/Web/HTML/Element/area

Reply:
Ok, calling it area now. (And recursively renamed all relevant arguments and functions)